cdserver/app/models/unit.rb

```diff
@@ -0,0 +1,6 @@
+# A room within a shelter building, especially for a family.   Units have more
+# than one bed, but those beds are not individually tracked within CARES.
```

Comment:
Question: can we tune how things are retrieved in memory?  I am guessing that it will be convenient to load the entire shelter structure into memory and leave it there.   We may even load it into Redis so it is available for all server instances.  This makes sense because the shelter structure almost never changes.    
I guess we can worry about that stuff later.